src/components/SeatsAnalysisViewer.vue

```diff
@@ -17,9 +17,9 @@
             <v-card-item class="d-flex justify-center align-center">
                 <div class="tiles-text">
                     <div class="text-overline mb-1" style="visibility: hidden;">filler</div>
-                    <div class="text-h6 mb-1">Assigned But Never Used</div>
+                    <div class="text-h6 mb-1">Assigned But Not Used</div>
                     <div class="text-caption">
-                        No show seats
+                        No show seats in the past 28 days
```

Comment:
@martedesco thanks for check and help. and for the seat API, it doesn't follow the 28 days limit, so if the assigned seat has never been used/or used 28 days ago, it also will be displayed here. so it is better to keep 'No show seats?' description? thanks.

Reply:
@DevOps-zhuang , updated [here](https://github.com/github-copilot-resources/copilot-metrics-viewer/pull/59)